src/components/OSCALControl.js

```diff
@@ -73,10 +73,18 @@ function ControlsList(props) {
 }
 
 export default function OSCALControl(props) {
+  if (
+    !props.includeAll &&
+    (!props.control ||
+      (props.includeControlIds &&
+        !props.includeControlIds.includes(props.control.id)))
+  ) {
+    return null;
+  }
   if (
     !props.control ||
-    (props.includeControlIds &&
-      !props.includeControlIds.includes(props.control.id))
+    (props.excludeControlIds &&
+      props.excludeControlIds.includes(props.control.id))
```

Comment:
I don't love how complex these if statements are getting, but I also am not sure of a better solution.